src-chrome-mv3/background.js

```diff
@@ -0,0 +1,46 @@
+console.log("Service-worker has loaded via background.js.");
+
+// The onClicked callback function.
+function onClickHandler(info, tab) {
+
+    console.log("item " + info.menuItemId + " was clicked");
+    console.log("info: " + JSON.stringify(info));
+    console.log("tab: " + JSON.stringify(tab));
+
+}
+
```

Comment:
The main difference between Firefox and Safari seems to be the `browser` versus `chrome` global.
Could we have a helper, e.g. `getBrowser()` that does some checking (`if (chrome !== undefined) { ...}`) and returns whatever will then be used as the browser object, if it otherwise has the same interface across browsers?